onnxruntime/python/tools/transformers/onnx_model_bert_tf.py

```diff
@@ -315,4 +421,5 @@ def remove_reshape_before_first_attention(self):
 
     def postprocess(self):
         self.remove_reshape_before_first_attention()
-        self.prune_graph()
+        # Temporary work around for the following comment as it will cause topological issues for a bert model
+        #self.prune_graph()
```

Comment:
I think that prune_graph() doesn't do anything wrong here, it deletes an unnecessary bunch of nodes. It's weird that it throws ORT error for this model by reporting a graph that's not existed any more. We probably want to accommodate for this model